mne/io/cnt/cnt.py

```diff
@@ -171,6 +171,7 @@ def read_raw_cnt(
     emg=(),
     data_format="auto",
     date_format="mm/dd/yy",
+    header="old",
```

Comment:
To make sure nobody accidentally passses `preload` by position accidentally to `header`
```suggestion
    *,
    header="old",
```